packages/rancher-tracing/charts/templates/deployment.yaml

```diff
@@ -0,0 +1,82 @@
+{{- if .Values.enabled }}
```

Comment:
Whats the point of values.enabled ? If you disable the sub-chart the whole thing should get skipped.